modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler.h

```diff
@@ -0,0 +1,141 @@
+#ifndef THERMOCYCLER_H
```

Comment:
This is a general comment about this file - objects should really not be instantiated in header files. They can be declared here (usually as pointers or whatever) but their definitions should really live in a `.cpp` or `.ino` file. We should also really consider whether these need to be global, though because of arduino's `setup` and `loop` style they may in fact need to be.